lib/vty.c

```diff
@@ -3320,13 +3320,13 @@ static void vty_mgmt_ds_lock_notified(uintptr_t lib_hndl, uintptr_t usr_data,
 	vty = (struct vty *)session_ctx;
 
 	if (!success) {
-		zlog_err("%socking for DS %u failed! Err: '%s'",
+		zlog_debug("%socking for DS %u failed! Err: '%s'",
```

Comment:
In this case it's an error though right?

Reply:
Yes it's still sn error. But since we are already throwing a error on vtysh through vty_out() I thought I will make that log_debug(), unless you still prefer a log_err().

Reply:
semantically, this is an error not a debug, so you must use `err`

Reply:
Yeah. I expected it so. Reverted them back to zlog_err() in the latest push.